runconfig/streams.go

```diff
@@ -107,3 +109,34 @@ func (streamConfig *StreamConfig) CloseStreams() error {
 
 	return nil
 }
+
+// CopyToPipe connects streamconfig with a libcontainerd.IOPipe
+func (streamConfig *StreamConfig) CopyToPipe(iop libcontainerd.IOPipe) {
+	copyFunc := func(w io.Writer, r io.Reader) {
+		streamConfig.Add(1)
+		go func() {
+			if _, err := io.Copy(w, r); err != nil {
```

Comment:
I wonder if we're here it makes sense to use `pools.Copy`, because each copy allocates 32Kb regardless of real copy size.